src/primitives/encode.rs

```diff
@@ -7,7 +7,7 @@
 //! prepending HRP strings etc.
 //!
 //! In general, directly using these adaptors is not very ergonomic, and users are recommended to
-//! instead use the higher-level functions at the root of this crate.
+//! instead use the higher crate level API.
```

Comment:
In 2013cf3b4d904da5efbdb5be3eb2e84e4c34f98e:

Do we want the word `higher` here? If so, it should be "higher-level".

Reply:
I copied the docs in `decode.rs`, wont me to change both?

Reply:
Will drop the "higher" altogther, here and in `decode` (from where I copied it).

Reply:
Done as a separate patch because of the two instances.